index-set-app/src/cmr/index_set/services/index_service.clj

```diff
@@ -26,13 +25,13 @@
 (defn gen-valid-index-name
   "Join parts, lowercase letters and change '-' to '_'."
   [prefix-id suffix]
-  (s/lower-case (s/replace (format "%s_%s" prefix-id suffix) #"-" "_")))
+  (string/lower-case (string/replace (format "%s_%s" prefix-id suffix) #"-" "_")))
```

Comment:
@sxu123's got an open PR that's using `safe-lowercase` for lots of things now ... maybe everything? Worth checking with her about ...

Reply:
We should not be calling safe-lowercase for things that don't allow nil.